autopush/websocket.py

```diff
@@ -520,6 +550,12 @@ def process_hello(self, data):
             connected_at=self.ps.connected_at,
             router_type=router_type,
         )
+        # NOTE: in tests, storing a value of None results in a corrupted
+        # return value of True. (This appears to have to do with a bug in
+        # either moto or boto in how it deals with values encoded to
+        # {"NULL": True}. For now, I'm avoiding storing values of None.
+        if wake_data is not None:
+            user_item['wake_data'] = wake_data
```

Comment:
Why not put this in the branch above that populates wake_data? (And just move the wake_data check after the user_item is made)